about.html

```diff
@@ -0,0 +1,148 @@
+                    <a class="email venue" href="#">m.adjiwanou@unesco.org</a>
+                </div>
+            </div>
+        </section>
+        <hr class="hr">
+        <section class="logoSection" id="logoSection">
+            <div class="container logoContainer">
+                <h2 class="logoHeading">UNESCO 2023 Logo</h2>
+                <hr class="hr">
+    
+                <p class="logoDescription">The UNESCO-Greece Melina Mercouri International 
+                    Prize is awarded every two years, and the winner, 
+                    who may be an individual, an institution or a non-governmental organization, 
+                    receives an award of US$30,000.
+                </p>
+                
+                <img class="logo-img" src="./images/unesco-logo.jpeg" alt="UNESCO-Logo">
+            </div>
+          </section>
+          <hr class="hr">
+          <section class="pastSection " id="pastSection">
+            <div class="inner-div logoContainer">
+                <h2 class="logo-section-heading">See the past UNESCO Summit</h2>
+                <hr class="hr">
+
+                <p class="pastDescription">Take a look at the last two UNESCO Events</p>
+    
+                <ul class="pastEvents">
+                    <li class="pastEvent event1">
+                        <h3>2022</h3>
+                        <h4>UNESCO 2022 Winners</h4>
+                    </li>
+                    <li class="pastEvent event2">
+                        <h3>2021</h3>
+                        <h4>UNESCO 2021 Winners</h4>
+                    </li>
+                </ul>
+            </div>
+          </section>
+
+          <section class="partners aboutPatSec" id="partners">
+            <div class="inner-div">
+
+                <h2>Partners</h2>
+
+                <hr class="hr">
+    
+                <ul class="partnersList">
+                    <li>
+                        <a href="https://www.microsoft.com"><img class="partnersListLogo" src="./images/personLogo1.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="https://www.google.com"><img class="partnersListLogo" src="./images/personLogo2.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="https://www.airbnb.com"><img class="partnersListLogo" src="./images/personLogo3.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="https://www.mozilla.org"><img class="partnersListLogo" src="./images/personLogo4.webp" alt=""></a>
+                    </li>
+                </ul>
+            </div>
+          </section>
+
+          <footer class="aboutPageFooter">
+            <a href="./index.html" class="logo"
+              ><img src="./images/unesco-logo.jpeg" alt="ICPC-Logo"
+            /></a>
+            <p>2023 | United Nations Educational, Scientific and Cultural Organization</p>
+          </footer>
```

Comment:
Let's learn how to use semantic HTML which is one of the [objectives](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Use%20semantic%20HTML%20tags.) for this project, we shouldn't have our footer tag in the main tag - [learn more....](https://www.w3schools.com/html/html5_semantic_elements.asp)